src/app/component/board/column/Column.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import ColumnData from "../../../structure/ColumnData";
+
+const Column = ({ columnData, currentPlayer, columnModified, hasWinner }): JSX.Element => {
```

Comment:
1. Mivel hosszútávon TypeScript-et is szeretnénk használni, ezért érdemes már most .tsx file-okkal dolgozni, és a JSX.Element-el
2. Újabban a React-ben ú.n. Functional components-ekkel érdemes dolgozni a Class components helyett, ezért a függvényeket és a komponenseket ennek megfelelően kell.
(mindenről küldök bővebb infót privátban)